CHANGES.md

```diff
@@ -1,4 +1,9 @@
-## Pending
+## [0.15.0] (2020-07-17)
```

Comment:
We should maybe update the date here. I'll merge this for now and publish the crates this afternoon.